OverlayPlugin.Core/EventSources/FFXIVRequiredEventSource.cs

```diff
@@ -233,13 +233,13 @@ struct PartyMember
             public int level;
             // @deprecated, please use partyType
             public bool inParty;
-            public long contentID;
+            public long contentId;
```

Comment:
FWIW, I'm not sure what is consistent for OP here.  There's https://github.com/OverlayPlugin/OverlayPlugin/issues/172 where we added `weaponId` right next to `WorldID`.

Reply:
This wasn't a "consistent with OP's overall style" adjustment, just a "consistent with other properties on this event" adjustment.